lib/matobjplist.gi

```diff
@@ -282,7 +261,7 @@ InstallMethod( \[\]\:\=, "for a plist vector, a positive integer, and an obj",
 InstallMethod( \{\}, "for a plist vector and a list",
   [ IsPlistVectorRep, IsList ],
   function( v, l )
-    return Objectify(TypeObj(v),[v![BDPOS],v![ELSPOS]{l}]);
+    return MakeIsPlistVectorRep(v![BDPOS], true, v![ELSPOS]{l});
```

Comment:
And this one...

Reply:
Sorry please ignore

Reply:
Yes.
In fact, the documentation of `\{\}` for vector objects does currently not state that the result shall be mutable if possible, but I think this should be added to the documentation.
(For matrix objects, there is already such a statement.)